lib/system/database.rb

```diff
@@ -145,6 +145,11 @@ def adapter_module
 ActiveSupport.on_load(:active_record) do
   if System::Database.oracle? && defined?(ActiveRecord::ConnectionAdapters::OracleEnhancedAdapter::DatabaseTasks)
     ActiveRecord::ConnectionAdapters::OracleEnhancedAdapter::DatabaseTasks.class_eval do
+
+      ActiveRecord::ConnectionAdapters::OracleEnhancedAdapter.permissions = ['unlimited tablespace', 'create session',
+                                                                             'create table', 'create view', 'create sequence',
+                                                                             'create trigger', 'create procedure']
```

Comment:
I don't understand where are these granted. But I think you need to add these changes to the main PR. And this one you can merge or suggest improvements.

Reply:
This is a new feature in 6.1: https://github.com/rsim/oracle-enhanced/pull/1885/files

Also, I think it's a breaking change because not all permissions are listed compared to those that were granted previously.

I thought this was what was causing the "insufficient privileges" error on Oracle, but now I think it's something else, and I'm investigating :dizzy_face: 
There is something weird about how the database configs are resolved, when there are both `url:` in the `database.yml` and `DATABASE_URL` set.

And by the way, I didn't push to your branch!!! I just pushed to `3scale:dependabot/bundler/rails-6.1.7.7`, I don't know why this commit appears on your PR!

Reply:
Ah, I know @akostadinov! I still didn't touch your branch, but I now have a theory how this commit got here: I pushed it to my branch first, and then I amended it. So, I guess your branch's base got diverged from the tip of the main branch.

Maybe you can rebase on top of the latest `dependabot/bundler/rails-6.1.7.7` (dropping `234d21c0d269c34ec13f7f3f1410a3d67f70ab2d`), so I can merge.